configs/testarchitect.json

```diff
@@ -9,7 +9,7 @@
   ],
   "selectors": {
     "lvl0": {
-      "selector": "#breadcrumb div",
+      "selector": "#sectionBreadcrumb div",
```

Comment:
@s-pace I'm trying to capture the combined text from group of links breadcrumb menu that's in a div with the class #sectionBreadcrumb. For example:
Automation Guide / Action-based testing language / The test language it's in the code as 
`<div class="sectionBreadcrumb">
<a href="https://docs.testarchitect.com/automation-guide/">Automation Guide </a>/  
<a href="https://docs.testarchitect.com/automation-guide/action-based-testing-language/">Action-based testing language </a>/ 
<a href="https://docs.testarchitect.com/automation-guide/action-based-testing-language/the-test-language/">The test language</a>
</div>` On this page https://docs.testarchitect.com/automation-guide/action-based-testing-language/the-test-language/ It doesn't seem to be working. Should the line be `"#sectionBreadcrumb a"`? Thanks

Reply:
The selector defined is wrong. `#` are used to target element by ID. Issues solved by https://github.com/algolia/docsearch-configs/commit/c789202c6b8e9d7d9587ec8de5e8a87328adc340